packages/cpp/README.md

```diff
@@ -102,6 +102,34 @@ There are two ways to configure clang-tidy's checks: through a Theia preference
     - for the .clang-tidy file: Checks: "-*,readability-*"
         - Meaning: disable all list-checks and enable all readability-* checks
 
+### Running clang-tidy as a task
+
+To be able to run clang-tidy as a task, you need to install the program.
```

Comment:
We should also mention that the `.clang-tidy` file is necessary when running `clang-tidy` as a task.
It will not work without one, or if you simply have the preferences set.